hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapLoaderTest.java

```diff
@@ -383,19 +376,9 @@ public String load(String key) {
 
         map.addInterceptor(new TestInterceptor());
 
-        try {
-            map.size();
-            // We can't expect that since the exception transmission in map-loader is heavily dependant on operation execution.
-            // See: https://github.com/hazelcast/hazelcast/issues/11931
-            // fail("Expected a NPE due to a null value in a MapLoader");
-        } catch (NullPointerException e) {
```

Comment:
It seems the test logic is modified in the new version so that we always expect an exception. However, the comment explicitly says that we shouldn't rely on this. @ahmetmircik, what was a reasoning to change the logic? Also, now the test is failing periodically https://github.com/hazelcast/hazelcast-enterprise/issues/3438

Reply:
I think, i was cleaning tests with wishful thinking.

The question i have is: Since we know that it is not working as expected and we have an issue for the root cause, should we still keep these tests to pretend as if it is working as expected? What about ignoring them and referencing them from the root cause issue?

***Issue for root cause is here: https://github.com/hazelcast/hazelcast/issues/11931

Reply:
@ahmetmircik, thanks for the update. I will ignore the tests and we fix an issue in 4.1